Tasks/DownloadSecureFile/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 127,
```

Comment:
Could you please change `127` to `132`?  We use our sprint number for this minor version (currently sprint 132).